metricbeat/module/kafka/_meta/kibana/7/visualization/944188f0-8e79-11e8-8fa2-3d5f811fbd0f-ecs.json

```diff
@@ -61,7 +61,7 @@
                         "split_color_mode": "rainbow",
                         "split_mode": "terms",
                         "stacked": "none",
-                        "terms_field": "kafka.consumergroup.topic",
```

Comment:
I see these fields present in the [field mapping](https://github.com/elastic/integrations/blob/main/packages/kafka/data_stream/consumergroup/fields/fields.yml). 
Do we need to do some changes there as well?

Reply:
Let me check if any of these need to dropped from integration field mapping. Also briefly check, if any obsolete field still used. Not planning to do major kafka package changes for now.

Reply:
Also updated for "kafka.consumergroup.partition -> kafka.partition.id".

The corresponding integration PR https://github.com/elastic/integrations/pull/4530 has many old filed mapping that are getting dropped.